docs/access/client.md

```diff
@@ -0,0 +1,240 @@
+
+
+## Minimal client configuration {: #minimal_configuration }
+
+Next to installing the CernVM-FS client, you should also create a minimal configuration file for it.
+
+This is typically done in `/etc/cvmfs/default.local`,
+which should contain something like:
+
+``` { .ini .copy }
+CVMFS_CLIENT_PROFILE="single"
+CVMFS_QUOTA_LIMIT=10000
+```
+
+More information on the structure of `/etc/cvmfs` and supported configuration settings is available
+[in the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-configure.html).
+
+### Client profile setting
+
+With `CVMFS_CLIENT_PROFILE="single"` we specify that this CernVM-FS client should:
+
+* Use the [proxy server](../appendix/terminology.md#proxy) specified via `CVMFS_HTTP_PROXY`, if that configuration setting is defined;
+* Directly connect to a [Stratum-1 replica server](../appendix/terminology.md#stratum1) that provides the repository being used if no proxy server
+  is specified via `CVMFS_HTTP_PROXY`.
+
+As an alternative to defining `CVMFS_CLIENT_PROFILE`, you can also set `CVMFS_HTTP_PROXY` to `DIRECT` to specify
+that no proxy server should be used by CernVM-FS:
+
+``` { .ini .copy }
+CVMFS_HTTP_PROXY="DIRECT"
+```
+
+We will get back to `CVMFS_HTTP_PROXY` later when [setting up a proxy server](proxy.md).
+
+### Quota limit for client cache
+
+The `CVMFS_QUOTA_LIMIT` configuration setting specifies the maximum size of the CernVM-FS client cache (in MBs).
+
+In the example above, we specify that no more than ~10GB should be used for the client cache.
+
+When the specified quota limit is reached, CernVM-FS will automatically remove files from the cache according
+to the [Least Recently Used (LRU) policy](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU),
+until half of the maximum cache size has been freed.
+
+The location of the cache directory can be controlled by `CVMFS_CACHE_BASE` if needed (default: `/var/run/cvmfs`),
+but **must** be a on a *local* file system of the client, not a network file system that can be modified by multiple
+hosts.
+
+Using a directory in a RAM disk (like `/dev/shm`) for the CernVM-FS client cache
+is possible but not recommended in general, mostly because the internal implementation
+in CernVM-FS to support this is pretty involved, and not extensively tested.
+
+For more information on cache-related configuration settings,
+[see the CernVM-FS documentation](https://cvmfs.readthedocs.io/en/stable/cpt-configure.html#cache-settings).
+
+
+## Completing the client setup
+
+To complete the setup of the CernVM-FS client component,
+a`cvmfs` user account and group must be added to the system,
+and the `/cvmfs` and `/var/lib/cvmfs` directories must be created.
+
+In addition, it is recommended to update the `autofs` configuration
+to enable auto-mounting of CernVM-FS repositories *(optional)*.
+
+All these actions can be performed in one go by running the following command:
```

Comment:
It's a bit unclear from here if this includes step below:
```{ .ini .copy }
OPTIONS="--timeout 3600"
```

Reading the other reviewer comments it's clear that it doesn't but from the text it feels like `"--timeout 3600"` could be a default of `cvmfs_config setup`

Reply:
This got reworded in e170c36, maybe that helps avoid the confusion.

If not, let's follow-up in a follow up PR

Reply:
Clearer now, looks good!